.example.env

```diff
@@ -0,0 +1,137 @@
+#2faSpec.ts
```

Comment:
I really like your idea of putting the name of the file in front of the variables here, keeps it much cleaner.